### PR TITLE
Clarify CLI instructions

### PR DESCRIPTION
**Pull request by sacrosanctic**
related conversation
https://discord.com/channels/457912077277855764/1298479734606528628

### CODE REVIEW

**Review by AdrianGonz97 — APPROVED** [review]
thanks!

File: packages/cli/commands/add/index.ts [hunk 1]
```diff
@@ -284,7 +284,7 @@ export async function runAddCommand(
 			.filter((a) => !!a);
 
 		const selected = await p.multiselect({
-			message: 'What would you like to add to your project?',
+			message: `What would you like to add to your project? ${pc.dim('(use arrow keys/space bar)')}`,
```
Comment (benmccann) [inline packages/cli/commands/add/index.ts:287]:
```suggestion
			message: `What would you like to add to your project? ${pc.dim('(use arrow keys / space bar)')}`,
```

**Comment by benmccann** [comment]
Do we need the same hint if you run `create`, etc ?

**Comment by AdrianGonz97** [comment]
this change is already shared with `create` since as it invokes `runAddCommand`

**Review by benmccann — CHANGES_REQUESTED** [review]
Oh, that's pretty weird. You'd want it to show up in the first create question - not the third

**Comment by AdrianGonz97** [comment]
it only shows up on the multi-select where there's a difference between `space` and `enter`. the others are just normal select prompts

**Comment by manuel3108** [comment]
I also understand it the way that the core problem is the multiselect. 
But I think we should be consistent and also add it to the tailwind plugin multiselect:

image

**Comment by AdrianGonz97** [comment]
I thought about that and figured that it's fine to exclude it from proceeding multi-select prompts as you've been "taught" what to do by the time you reach this prompt

**Comment by sacrosanctic** [comment]
What is the reason for not adding hints to subsequent multi-selects? Is it because of clutter?

Isn't it always better to provide visual aid, I understand it's squares for multi-select and circles for single-select. But I find that hardly noticeable.

**Comment by benmccann** [comment]
I'd have to see what it looks like, but my first reaction is that it'd be too noisy and we should mention it only once

**Comment by AdrianGonz97** [comment]
seeing it now, I still think it adds unnecessary noise. this noisiness is also exacerbated when the instructions are kept on screen long after prompt submission

**Comment by sacrosanctic** [comment]
Is it possible to add a check to only show the hint on the first multi-select  the user encounters, or would that increase the code complexity.

If possible, this would reduce clutter and still have hints regardless of which branch the user takes.

**Review by manuel3108 — APPROVED** [review]

**Comment by manuel3108** [comment]
Let's only add this to the integration-selection question for now, as everyone seems to agree there. Let's get this merged and hear for user feedback

**Pull request merged**